Tasks/DownloadPackageV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -22,5 +22,6 @@
   "loc.messages.RetryingOperation": "Error: in %s, so retrying => retries pending: %s",
   "loc.messages.OperationFailed": "Failed in %s with error: %s",
   "loc.messages.Info_ResolvedToolFromCache": "Resolved from tool cache: %s",
-  "loc.messages.UnsupportedProjectScopedFeeds": "Project-scoped feeds are not supported by this version of the \"Download Package\" task. Please use version 1 or higher."
+  "loc.messages.UnsupportedProjectScopedFeeds": "Project-scoped feeds are not supported by this version of the \"Download Package\" task. Please use version 1 or higher.",
+  "loc.messages.DeprecatedTask": "This task is now deprecated. The task completed successfully, but builds are breaking. Functionality will stop on 11-27-2023. Please switch to using DownloadPackage@1 as soon as possible."
```

Comment:
Confusing to say "task completed successfully". "Builds are breaking" is also a potentially internal term. I think we should just say "This task is deprecated. Please replace this task with DownloadPackage@1" and link to downloadpackage@1 docs

Reply:
I get how that can be confusing. What I meant is that the error is thrown at the end for now, so it's possible to still use the task, but users should really move onto alternatives, and after 1/16/24 they will have to. Let me know if the message I changed it to is less confusing

Reply:
Using Matt's version!